### PR TITLE
CSPL-2052: merge azure blob appframework feature to develop

### PR DESCRIPTION
**Pull request by kumarajeet**
Merge PR for bringing the appframework azure blob feature work to the develop branch.

### CODE REVIEW

File: .github/workflows/int-test-workflow.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - master
+      - appframework_azureblob
```
Comment (sgontla) [inline .github/workflows/int-test-workflow.yml:7]: nit: do we still need this?

**Comment by sgontla** [comment]
Seems like the CRDs are missing the probes related changes. Can you rebuild the CRDs and update the PR?

**Comment by sgontla** [comment]
> changes

Please ignore this comment.

File: .github/workflows/int-test-workflow.yml (hunk 1 above)
Comment (kumarajeet) [reply .github/workflows/int-test-workflow.yml]: Removed.

**Review by sgontla — APPROVED** [review]

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -129,6 +131,7 @@ If you need help, tag one of the active maintainers of this project in a post or
 (*) Arjun Kondur
 (*) Kriti Ashok
 (*) Param Dhanoya
+(*) Vivek Reddy
 () Victor Ebken
 () Ajeet Kumar
 () Marcus Galdino
```
[the remark this answers is not in the document]
Comment (kumarajeet) [reply docs/CONTRIBUTING.md:137]: whole maintainers section?

File: .github/workflows/int-test-workflow.yml [hunk 3]
```diff
@@ -95,6 +89,7 @@ jobs:
       S3_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
       ENTERPRISE_LICENSE_LOCATION: ${{ secrets.ENTERPRISE_LICENSE_LOCATION }}
       CLUSTER_WIDE: "true"
+      DEPLOYMENT_TYPE: "helm"
```
Comment (pdhanoya-splunk) [inline .github/workflows/int-test-workflow.yml:92]: We need to remove this line.

**Comment by gaurav-splunk** [comment]
We will fix all the remaining comments in a separate PR in develop branch.

File: docs/CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gaurav-splunk) [reply docs/CONTRIBUTING.md:137]: Only Marcus's name

**Review by gaurav-splunk — APPROVED** [review]

**Pull request merged**